implants/golem/tests/cli.rs

```diff
@@ -22,26 +22,26 @@ fn test_golem_main_file_not_found() -> anyhow::Result<()> {
     Ok(())
 }
 
-// Test running `./golem ../../tests/golem_cli_test/syntax_fail.tome`
+// Test running `./golem ../../bin/golem_cli_test/syntax_fail.tome`
 #[test]
 fn test_golem_main_syntax_fail() -> anyhow::Result<()> {
     let mut cmd = Command::cargo_bin("golem")?;
 
-    cmd.arg("../../tests/golem_cli_test/syntax_fail.tome");
+    cmd.arg("../../bin/golem_cli_test/syntax_fail.tome");
```

Comment:
it might make sense abstracting out the dir to a global test var, so its easy to change in the future.

Reply:
I agree.
Unfortunately that doesn’t work for everything like the include_bytes macro 😭.
I’ll try changing it where I can but would like to maybe write a wrapping macro that takes in a var.